app/utils/kcidb.py

```diff
@@ -154,6 +154,13 @@ def push_tests(group_id, bq_options, db_options={}, db=None):
     ns = bq_options.get("namespace", "kernelci.org")
     test_cases = _get_test_cases(group, db, [], ns)
     build = _get_build_doc(group, db)
+    if not build:
+        utils.LOG.warn("kcidb: Missing build, unable to push tests.")
+        build = {
```

Comment:
What's the point of using fake build IDs?  We should either change the schema if we don't require a build and put nothing here, or not push test results if a build is required.

Reply:
This patch is marked `TEST; DO NOT MERGE` it's just useful for testing the code on a backend instance with no builds in storage.

Reply:
Sorry that didn't appear in the PR title.

Reply:
Only the 3rd patch had that title.  The first 2 patches were for merge consideration (but would need resconsideration depending on https://github.com/kernelci/kernelci-backend/pull/187)